modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Instructions.kt

```diff
@@ -36,22 +36,28 @@ const val CAN_BURN_ASSET_WITH_DEFINITION = "can_burn_asset_with_definition"
 const val ASSET_ID_TOKEN_PARAM_NAME = "asset_id"
 const val ASSET_DEFINITION_PARAM_NAME = "asset_definition_id"
 
-// additional support for fixed types
-// register peer
-// burn
-// fail
-// grant
-// if
-// mint
-// pair
-// register
-// remove key value
-// sequence
-// set key value
-// transfer
-// unregister
+/**
```

Comment:
Please remove it. I left it just a reminder what instructions need to implement. As far as you made it, no need in tis reminder